ut/s3_object_metadata_test.cc

```diff
@@ -46,7 +45,7 @@ class S3ObjectMetadataTest : public testing::Test {
     evhtp_request_t *req = NULL;
     EvhtpInterface *evhtp_obj_ptr = new EvhtpWrapper();
     call_count_one = 0;
-    bucket_name = "seagatebucket";
+    bucket_name = "seagate_bucket";
```

Comment:
In tests requests inside payload have metadata attribute "seagate_bucket" instead of "segatebucket". It revelaed during development of the patch. Had a talk with @dsurnin and it was proposed to leave this change in the patch.